storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -2210,9 +2212,17 @@ decrypt_decompress_file(const char *filepath, uint thread_n)
  		message << "decompressing";
  		needs_action = true;
  	}
-
-	cmd << " > " << SQUOTE(dest_filepath);
- 	message << " " << filepath;
+	message << " " << filepath;
+        output_file << "xboutputfile" << thread_n;
```

Comment:
it would be nice to use capitalnames for environmental variables.

Reply:
I choose small letter environmental variables because OS   Uses capital letter and it doesn't interfere with them